testIssues.json

```diff
@@ -0,0 +1,150 @@
+{
```

Comment:
I don't think you wanted to commit this file? Or do you actually have some tests for the CLI that use this test data?